libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -145,7 +146,13 @@ class MapboxNavigation(
     private val fasterRouteController: FasterRouteController
     private val routeRefreshController: RouteRefreshController
     private val arrivalProgressObserver: ArrivalProgressObserver
-    private val navigatorConfig = NavigatorConfig(null, null, null)
+    private val electronicHorizonOptions: ElectronicHorizonOptions = ElectronicHorizonOptions(
+        navigationOptions.eHorizonOptions.length,
+        navigationOptions.eHorizonOptions.expansion.toByte(),
+        navigationOptions.eHorizonOptions.branchLength,
+        navigationOptions.eHorizonOptions.includeGeometries
+    )
+    private val navigatorConfig = NavigatorConfig(null, electronicHorizonOptions, null)
```

Comment:
Set `EHorizonOptions` from `NavigationOptions` to `NavigatorConfig` https://github.com/mapbox/mapbox-navigation-android/pull/3748/commits/3ed960fd8a370d392a9f690cf0d23da031d80621 Great catch @kmadsen